### ISSUE TITLE
release 2.5.0 - boost preprocessor not found

### ISSUE THREAD

**Issue by ajaypanyala** [evaleev/libint#121]
Trying out release 2.5.0 pre-generated lib (std ints) - config.log [here](https://github.com/evaleev/libint/files/3051021/config.log)

I get an error when compiling hartree_fock++

```
 /opt/libint/include/libint2/engine.impl.h:35:11: fatal error: boost/preprocessor.hpp: No such file or directory
 # include <boost/preprocessor.hpp> 
           ^~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by evaleev** [comment]
Could you please post the config.log file? Looks like usable system boost
should have been found

**Comment by ajaypanyala** [comment]
https://github.com/evaleev/libint/files/3051021/config.log (already included in the initial post)

**Comment by evaleev** [comment]
fixed via 4c5efd1527d7ea82e4d47edbeeab9230ac353092 ... I pulled 2.5.0 and will rerelease today ... thanks for the heads up!
